image-config.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   #Versions places here for gradle will be used generate the relevant dockerfiles
   #after you run generate.sh
   builders:
-    jdk7:
-      tag: jdk:7,maven:3.8,ant:1.9.16
-    jdk17:
+    ubi7:
+      tag: jdk:7,maven:3.8.8,ant:1.9.16
```

Comment:
I have had to push this change separately, as the JDK7 build was broken without it.